upi/vsphere/main.tf

```diff
@@ -1,95 +1,97 @@
 provider "vsphere" {
-  user                 = "${var.vsphere_user}"
-  password             = "${var.vsphere_password}"
-  vsphere_server       = "${var.vsphere_server}"
+  user                 = var.vsphere_user
+  password             = var.vsphere_password
+  vsphere_server       = var.vsphere_server
   allow_unverified_ssl = true
 }
 
 data "vsphere_datacenter" "dc" {
-  name = "${var.vsphere_datacenter}"
+  name = var.vsphere_datacenter
 }
 
 module "folder" {
   source = "./folder"
 
-  path          = "${var.cluster_id}"
-  datacenter_id = "${data.vsphere_datacenter.dc.id}"
+  //path          = "${var.cluster_id}"
+  path          = "Dani/${var.vm_folder}"
```

Comment:
Is 'Dani/' prepended to the folder path why the tests fail? You probably don't want this included in the PR?

Reply:
thanks @fherbert , fixed